websites/N/Nix Mangas/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Luscarvalho",
+		"id": "458776046184169473"
+	},
+	"service": "Nix Mangas",
+	"description": {
+		"en": "Nix Mangas is a site to read manga and manhwas without ads",
+		"pt_BR": "Nix Mangás é um site para ler mangás e manhwas sem anúncios"
+	},
+	"url": "nixmangas.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/7ooKWuO.png",
```

Comment:
Your logo should be [512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo)